### DISCUSSION TITLE
how to upload trained model from gdrive to hugging face

### DISCUSSION THREAD

**Discussion by BigDogg99** [TheLastBen/fast-stable-diffusion#864] [Q&A] [answered]
title

**Comment by TheLastBen** [comment, marked as answer]
There is a cell to do that after the testing cell

**Reply by BigDogg99** [reply]
is there a way to upload it on hugging face if i already have a have a trained model in my gdrive

**Reply by TheLastBen** [reply]
yes, create a session, then put the ckpt in the session's folder, make sure you rename it to the session's name, then reload the session and it will convert it to diffusers, then you can upload it to hugging face (make sure you run the testing cell for a few seconds)

**Reply by BigDogg99** [reply]
thanks i was able to do it with cpkt file but .safetensor file not working
